front/app/containers/ProjectsShowPage/timeline/Ideas.tsx

```diff
@@ -91,14 +90,15 @@ const IdeasContainer = ({ projectId, phase, className }: InnerProps) => {
       {isVotingContext ? (
         <IdeaCardsWithoutFiltersSidebar
           defaultSortingMethod={ideaQueryParameters.sort}
+          invisibleTitleMessage={messages.a11y_titleInputsPhase}
           showDropdownFilters={false}
           showSearchbar={false}
           {...sharedProps}
         />
       ) : (
         <>
           <IdeaListScrollAnchor />
-          <IdeaCardsWithFiltersSidebar inputTerm={inputTerm} {...sharedProps} />
+          <IdeasWithFiltersSidebar inputTerm={inputTerm} {...sharedProps} />
```

Comment:
If you don't wrap this with a `Suspense`, lazy loading won't work as intended. It will fall back to a `Suspense` higher up the tree, the one at the page level in routes.tsx (this fallback is also the reason it is currently not crashing). So it'll just be rendered asap and not let other components go first. 

(Can be verified with a simple console.log in IdeasWithFiltersSidebar. When it's wrapped with Suspense, the log arrives noticeably later.)